src/Resources/config/view_cache.yml

```diff
@@ -41,3 +50,8 @@ services:
         class: EzSystems\PlatformHttpCacheBundle\ResponseTagger\Value\LocationTagger
         tags:
             - {name: ezplatform.cache_response_tagger}
+
+    ezplatform.view_cache.response_tagger.restlocationbc:
+        class: EzSystems\PlatformHttpCacheBundle\ResponseTagger\Value\RestLocationBCTagger
+        tags:
+            - {name: ezplatform.cache_response_tagger}
```

Comment:
I really don't get the reasoning behind splitting the response taggers in delegators and values. It just creates overhead? Also, given that we keep this split, why does also the value tagges have the `ezplatform.cache_response_tagger` tag? It means they are both invoked by the dispatcher directly, and by their delicator